packages/browser-sdk/src/auto-confirm/getProvider.ts

```diff
@@ -0,0 +1,15 @@
+import type { PhantomProvider } from "./types";
+
+export function getProvider(): PhantomProvider | null {
+  if (typeof window === "undefined") {
+    return null;
+  }
+
+  const provider = (window as any).phantom?.app
+
+  if (!provider) {
```

Comment:
```suggestion
  if (!provider || !provider?.isPhantom) {
```

Reply:
I don't think it's necessary it won't find phantom if it wasn't phantom. If it's an impostor isPhantom doesn't make much difference.